ibazel/command/command.go

```diff
@@ -86,3 +97,27 @@ func subprocessRunning(cmd *exec.Cmd) bool {
 
 	return true
 }
+
+func terminate(pg process_group.ProcessGroup) {
+	pg.Signal(syscall.SIGTERM)
+	done := make(chan bool, 1)
+	go func() {
+		select {
+		case <-time.After(*gracefulDuration):
+			log.Logf("The subprocess wasn't terminated within %s", *gracefulDuration)
```

Comment:
Can you add information about what is going to happen now that it wasn't terminated within %s?

Something like "The subprocess wasn't terminated within %s and was killed forcefully?" Of course that would require putting it on the line after `sendKillSignal`

Reply:
Currently logs look like this:

```
iBazel [4:43PM]: The subprocess wasn't terminated within 10s
iBazel [4:43PM]: Sending SIGKILL to the subprocess

iBazel [4:43PM]: Subprocess killed from getting SIGINT (trigger SIGINT again to stop ibazel)
```

To me it feels important having logs in chronological order. First comes a timeout, then a signal and then at some point the subprocess goes down.

How about "The subprocess wasn't terminated within %s. Forcing to close"?

Then it would be:

```
iBazel [4:58PM]: The subprocess wasn't terminated within 10s. Forcing to close.
iBazel [4:58PM]: Sending SIGKILL to the subprocess

iBazel [4:58PM]: Subprocess killed from getting SIGINT (trigger SIGINT again to stop ibazel)
```

What do you think?

Reply:
You're right. When considering the log output holistically your choice is superior.